application/managers/http/manager.py

```diff
@@ -0,0 +1,75 @@
+"""
+Manager for http business logic.
+"""
+
+
+import httpx
+from fastapi import Depends
+
+from constants.events import EventCategory
+from exceptions.http import HttpException
+from managers.events import EventManager
+from managers.events import get_event_manager
+from managers.kubernetes import K8sManager
+from managers.organizations.manager import OrganizationManager
+from managers.organizations.manager import get_organization_manager
+from models.organization import Organization
+from schemas.events import EventSchema
+from services.helm.facade import HelmService
+
+
+class HttpManager:
+    """
+    Manager to manipulate Http requests.
+    """
+    organization_manager: OrganizationManager
+
+    def __init__(self, organization_manager: OrganizationManager, event_manager: EventManager):
+        self.organization_manager = organization_manager
+        self.event_manager = event_manager
+
+    ############################################################################
+    # Http Requests
+    ############################################################################
+
+    async def http_request(
+        self, component_name: str, url: str, method: str,
+        parameters: dict | list, headers: dict = None, dry_run: bool = False
+    ) -> str:
+        """
+        Send an HTTP request to the specified URL.
+        """
+        # Convert to a dict if a list of dicts provided
+        if isinstance(parameters, list):
+            parameters = {k: v for d in parameters for k, v in d.items()}
+
+        # Create an HTTP client
+        async with httpx.AsyncClient() as client:
+            # Prepare the request
+            method = method.lower()
+            match method:
```

Comment:
Please no magic string. Refer to items from `ComponentTypeHttpMethods` enum. Or better use `httpx.request` get rid of this match-case statement. At this point here should be only HTTP method allowed in component model.

Reply:
[Docs](https://www.python-httpx.org/api/) about httpx.request.

Reply:
This allows the user to use UPPERCASE or lowercase in the tempalte.